pkg/cli/admin/update/update.go

```diff
@@ -52,64 +52,77 @@ func NewOptions(streams genericclioptions.IOStreams) *Options {
+			Passing --to=VERSION or --to-image=IMAGE will update the cluster to one of the available
+			updates or report an error if no such version exists. The cluster will then update
 			itself and report status that is available via "oc get clusterversion" and "oc describe
 			clusterversion".
 
-			Passing --to-multi-arch will upgrade the cluster from a single-architecture to a
+			Passing --to-multi-arch will update the cluster from a single-architecture to a
 			multi-architecture cluster at the current version.
 
 			If there are no versions available, or a bug in the cluster version operator prevents
 			updates from being retrieved, --to-image may be combined with the more powerful and
-			dangerous --allow-explicit-upgrade. This instructs the cluster to upgrade to the contents
-			of the specified release image, regardless of whether that upgrade is known to be
+			dangerous --allow-explicit-update. This instructs the cluster to update to the contents
+			of the specified release image, regardless of whether that update is known to be
 			recommended for the current version. While rolling back to a previous patch (z stream) version
-			(4.1.2 -> 4.1.1) may be safe, upgrading more than one minor version ahead (4.1 -> 4.3) or
+			(4.1.2 -> 4.1.1) may be safe, updating more than one minor version ahead (4.1 -> 4.3) or
 			downgrading one minor version (4.2 -> 4.1) is likely to cause data corruption or to
 			completely break a cluster.
 
-			There are two layers of upgrade guards: client-side and cluster-side.
+			There are two layers of update guards: client-side and cluster-side.
 
-			Client-side guards include checks for whether the cluster is already being upgraded, or if
+			Client-side guards include checks for whether the cluster is already being updated, or if
 			the cluster is reporting a failure.  It is usually best to give these conditions time to
 			resolve, or to actively work to resolve them.  But if you decide to trigger the update
-			regardless of these concerns, use --allow-upgrade-with-warnings.
+			regardless of these concerns, use --allow-update-with-warnings.
 
-			Cluster-side guards include checks for release verification and upgradeable conditions.
+			Cluster-side guards include checks for release verification and updateable conditions.
 			Again, it is usually best to give these conditions time to resolve, or to actively work to
 			resolve them.  But if you decide to trigger the update regardless of these concerns,
 			use --force, which is passed through to ClusterVersion's spec.desiredUpdate.force.
 		`),
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if cmd.CalledAs() != "update" {
+				fmt.Fprintf(o.ErrOut, "%s is deprecated.  Use '%s ...' instead\n", cmd.CommandPath(), cmd.CommandPath())
+			}
+			return nil
+		},
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(o.Complete(f, cmd, args))
 			kcmdutil.CheckErr(o.Run())
 		},
+		Aliases: []string{"upgrade"},
```

Comment:
:+1:

Reply:
Do we need to add some code to warn people saying `upgrade` is deprecated and they should start using `update` even if we will never remove `upgrade` alias.

Reply:
Deleted my previous comment about the warning message about `upgraded` because I saw that we already printing the message.